### ISSUE TITLE
Specifically ban the use of trap against non-trap trans people/characters

### ISSUE THREAD

**Issue by nahweh** [0-Shine-0/Goodanimemes-Rules#10]
I do think we should codify the Komi subreddit's approach and specifically ban the use of "trap" towards trans people under the "no hate speech" section. A lot of Reddit has had eyes on the whole trap ban debacle and a lot of users see this sub as a hub of transphobia. I think it'd behoove us to show Reddit and trans people that we are eager to meet them halfway in making them feel as welcome as possible without compromising on the integrity of anime meme culture. Not only would it be a good chance to dispel accusations of transphobia against our community, but it would also be a good chance to show Animemes mods how tactful moderation is really done, especially since the Komi sub approach was argued for by our community a lot during the debacle with Animemes.

**Comment by teisseire117** [comment]
No in my opinion we should let this drama behind us.
Those who call us transphobic are clearly hateful people who do not care about reality. You could say whatever you want : 1/ They won't read it 2/ if they read it, they'll say you're transphobic in disguise.

What's the point ? I do not like the idea to try to appease your bully, let's just ignore them.
Misuse of the trap word is already punished and do not need further explanation.

**Comment by edenprojectde** [comment]
> No in my opinion we should let this drama behind us.
> Those who call us transphobic are clearly hateful people who do not care about reality. You could say whatever you want : 1/ They won't read it 2/ if they read it, they'll say you're transphobic in disguise.
> 
> What's the point ? I do not like the idea to try to please your bully, let's just ignore them.
> Misuse of the trap word is already punished and do not need further explanation.

it will at least take wind out of their sail, and i even think we should clarify the meaning of the word trap as well too

**Comment by 0-Shine-0** [comment]
I do believe we should take this approach.
There's nothing to gain by misrepresenting characters, we won't ban the word, but will regulate it's use. As it should have been.
So in cases like Hideri from Blend-S, you can say trap, but not with Felix for example, as she's pretty much the textbook definition of transexual. 
We won't ban if it's just an honest mistake, just remove the post and done.

**Comment by edenprojectde** [comment]
Felix from Re:Zero?
His gender is male everywhere 
![image](https://user-images.githubusercontent.com/6115966/92296660-51a18100-ef37-11ea-8733-3ea7dded6f06.png)
![image](https://user-images.githubusercontent.com/6115966/92296670-5b2ae900-ef37-11ea-95a4-ba0af22ee8f4.png)
![image](https://user-images.githubusercontent.com/6115966/92296675-6b42c880-ef37-11ea-88c4-50ae6c2e98cc.png)

**Comment by teisseire117** [comment]
Yeah the felix case is only used by the drama queens who, again, don't care about reality, don't try to get to know it, they just hating for no reasons without planning to stop.
That's why we should just not fuel this senseless drama again, it makes no sense.

**Comment by edenprojectde** [comment]
@teisseire117 it really just makes sense to have a glossary(defining our language, like trap and other missrepresented words), 

also **the topic isnt driven by the amount of cases but rather ignorance from some authorian trans/lgbtq+ people** who think we want to ruin trans peoples existence. (even though nobody in the community who is active on here shows any sign of being like that, and most even either being the nicest persons they can be around others and are themselves not even straight for the most part, which is then again just ignorance from authorian trans/lgbtq+ people)

important to note: im not implying all trans/lgbtq+ are authorian, im just talking about these specific people.

<sub>Sent with <a href="http://githawk.com">GitHawk</a></sub>

**Comment by edenprojectde** [comment]
I also think the main point of making this stated as a rule, is to just stop the wind in their sails, as they spout these claims to other people as well.

<sub>Sent with <a href="http://githawk.com">GitHawk</a></sub>
